atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -0,0 +1,112 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+
+create a group of the image manifests from the worker builds
+"""
+
+
+from __future__ import unicode_literals
+from copy import deepcopy
+import requests
+import requests.auth
+import json
+
+try:
+    from urlparse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
+
+from atomic_reactor.plugin import PostBuildPlugin
+from atomic_reactor.plugins.build_orchestrate_build import OrchestrateBuildPlugin
+from atomic_reactor.util import Dockercfg
+
+
+class GroupManifestsPlugin(PostBuildPlugin):
+    key = 'group_manifests'
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow, registries, group=True, goarch={}):
```

Comment:
Avoid using mutable objects as default values. Instead, set it to `None` and create a new instance in method body.

```python
..., goarch=None):
  goarch = goarch or {}
```